sysdeps/nacl/execv.c

```diff
@@ -0,0 +1,17 @@
+#include <errno.h>
+#include <unistd.h>
+#include <stub-tag.h>
+
+/* Execute PATH with arguments ARGV and environment from `environ'.  */
+int
+__libc_execv (const char *path, char *const argv[])
+{
+  int retval;
+  retval = 500;
```

Comment:
So here the return value of 500 was set by me for test purpose only. According to the Linux man page, the execv() function only returns if an error has occurred, and the return value should be set to -1.

Reply:
Gotcha, will change that.